CDCgov/prime-reportstream
# 1139 Add HL7 ingest optional field test and code coverage

This PR adds a test to the HL7 ingest for optional fields and enables code coverage reports.

Test:
1. Build the baseline with `./gradlew package`
2. Verify the Observation tests pass.
3. Open build\reports\jacoco\test\html\index.html and verify it contains code coverage information

## Changes
- adds a test to the HL7 ingest for optional fields
- enables code coverage reports

## Checklist

### Testing

### Security

### Process

## Security
*List potential security threats and mitigations in the PR*

## Fixes
- #1139

## To Be Done
